REST API Changes (#139)
lmstudio-ai/docs: 1_python/1_getting-started/_authentication.md

index: 3
---

> LM Studio, from version 0.4.0, supports API Tokens for authentication, providing a secure and convenient way to access the LM Studio API.

Reviewer: ```suggestion
##### Requires [LM Studio 0.4.0](/download) or newer.

LM Studio supports API Tokens for authentication, providing a secure and convenient way to access the LM Studio API.
```
